### DISCUSSION TITLE
Documentation Update

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [reactplay/react-play#145] [Ideas]
Hello @atapas sir should we add a new readme which displays the best plays we have on the website and we'll add the link to it on our main readme of our project

And if you think we should implement the idea then I'll start working on it

**Comment by atapas** [comment]
> Hello @atapas sir should we add a new readme which displays the best plays we have on the website and we'll add the link to it on our main readme of our project
> 
> And if you think we should implement the idea then I'll start working on it

It is a good idea but, it shouldn't be in the readme or any document. Instead, it should be part of the application. Something like trending plays and leaderboard for the creators.

I have starred a discussion on it yesterday. Please post your thoughts.

**Reply by atapas** [reply]
https://github.com/atapas/react-play/discussions/142

**Reply by atapas** [reply]
Let me know what do you think.

**Reply by hasnainmakada-99** [reply]
Ya I think its also a great idea tapas sir

**Reply by atapas** [reply]
@hasnainmakada-99 Should we lock this discussion then?

**Reply by hasnainmakada-99** [reply]
Yeah sure
